Assets/Code/Scripts/PlayerManagement/MouseLook.cs

```diff
@@ -52,12 +67,26 @@ void Update()
             _yRotation += mouseX;
             _xRotation -= mouseY;
             _xRotation = Mathf.Clamp(_xRotation, -90f, 90f);    // Clamping allows to block the rotation
-        
-            // rotate camera and orientation
-            /*This order is needed because if the object is the same, the second assign value overwrites the values of the first one.
```

Comment:
Insert this comment in one or both the if statement, I have left this comment because this stupid order caused me some trouble

Reply:
I left it there because I was way out of reach with the current PRs. Nicola said the inverted Y doesn't work either, so I'll definitely have a look into that.

Reply:
Comment deleted, it should be clearer now.